api/files/serializers.py

```diff
@@ -336,6 +340,17 @@ class FileDetailSerializer(FileSerializer):
     id = IDField(source='_id', required=True)
 
 
+class QuickFilesSerializer(BaseFileSerializer):
+    user = RelationshipField(related_view='users:user-detail',
+                             related_view_kwargs={'user_id': '<node.creator._id>'},
```

Comment:
@erinspace Can you make sure that accessing `node.creator._id` here isn't resulting in a lot of added queries? If so, we might need to do some prefetching or .include-ing.